prow/plugins/approve/approvers/README.md

```diff
@@ -154,7 +156,7 @@ K8s-bot updates comment:
 	
 	[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
-	This pull-request has been approved by: *approver1, PRAuthor*
+	This pull-request has been approved by: *approver1, approver3, PRAuthor*
```

Comment:
Similarly, why is `approver3` added here and below?

Reply:
See above; I tried to actually read through the documentation, and it appears that both "Case a" and "Case c" (at least) are required to reach "Step 3"

I'm happy to rewrite this however needed to make the behavior clear.